### PR TITLE
[ONNX] List of files to consider for mergebot onnx rule

### PR DESCRIPTION
**Pull request by BowenBao**
Based on past PRs, here is an non-exhaustive list of files to consider for extension. The PR is not meant to be final. Based on feedback and discussion, files could be dropped from the list, or PR could be updated to move code around such that extension is no longer needed.

List of files below and description:

* These files are for converting from IR to ONNX proto. These should be used only for ONNX.
```
"torch/csrc/jit/serialization/export.*",
"torch/csrc/jit/serialization/onnx.*",
```

* This file is touched whenever pass signature is updated.
```
"torch/_C/__init__.pyi.in",
```

* These files are touched whenever pass signature is updated. Somehow it's been convention that onnx passes are also added here, but it could be possible to move them. Let me know what you think.
~~"torch/csrc/jit/python/init.cpp",~~
~~"torch/csrc/jit/python/script_init.cpp",~~
Update: Bowen will move onnx passes to files under onnx folder.

* ~~Touched when need new attr::xxx, or onnx::xxx.~~
~~"aten/src/ATen/core/interned_strings.h"~~
Update: Nikita will help separate this file.

@malfet

### CODE REVIEW

File: torch/csrc/jit/python/script_init.cpp [hunk 1]
```diff
@@ -2206,10 +2206,6 @@ void initJitScriptBindings(PyObject* module) {
       logging::LoggerBase,
       std::shared_ptr<logging::NoopLogger>>(m, "NoopLogger")
       .def(py::init<>());
```
Comment (garymm) [inline torch/csrc/jit/python/script_init.cpp:2208]: what's the difference between python/script_init and python/init?

**Review by garymm — COMMENTED** [review]
Remove mention of `interned_strings` from PR description.

File: torch/csrc/jit/python/script_init.cpp (hunk 1 above)
Comment (BowenBao) [reply torch/csrc/jit/python/script_init.cpp:2208]: init: jit bindings
script_init: jit script bindings

**Review by garymm — APPROVED** [review]

**Comment by BowenBao** [comment]
@malfet please review and import.

**Comment by BowenBao** [comment]
@malfet friendly ping for updates

**Review by malfet — APPROVED** [review]

**Comment by garymm** [comment]
@malfet please merge

**Pull request closed without merging**